native/explorer/src/series.rs

```diff
@@ -53,32 +53,38 @@ from_list!(s_from_list_str, String);
 macro_rules! from_list_float {
     ($name:ident, $type:ty, $module:ident) => {
         #[rustler::nif(schedule = "DirtyCpu")]
-        pub fn $name(name: &str, val: Term) -> ExSeries {
+        pub fn $name(name: &str, val: Term) -> NifResult<ExSeries> {
```

Comment:
Is it necessary to use Rustler's NifResult here? Do you know if the `Result<ExSeries, ExplorerError>` could be used instead?

Reply:
Ok, thanks!